packages/-ember-data/yuidoc.json

```diff
@@ -3,7 +3,7 @@
   "description": "The ember-data API: a data persistence library for Ember.js",
   "url": "https://github.com/emberjs/data",
   "options": {
-    "enabledEnvironments": ["production"],
+    "enabledEnvironments": [],
```

Comment:
What do `enabledEnvironments` do?

Reply:
if you add an environment the docs will build when a build or serve for that environment occurs. This was the root of the reason why we were doing production builds during prepublish before, now we just call the docs script directly.

Reply:
:+1: